loadtest/loadtest.go

```diff
@@ -305,6 +324,7 @@ func New(config *Config, nc NewController, log *mlog.Logger) (*LoadTester, error
 		status:            Status{},
 		activeControllers: make([]control.UserController, 0),
 		idleControllers:   make([]control.UserController, 0),
+		isBrowserAgent:    isBrowserAgent,
 		log:               log,
```

Comment:
Adding the comment here because I cannot add it to the specific line (322), but `statusChan` should be something like
```go
statusChan: make(chan control.UserStatus, config.UsersConfiguration.MaxActiveUsers + config.UsersConfiguration.MaxActiveBrowserUsers),
```